src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -26,10 +26,11 @@ your environment (default credentials: admin/admin) or just use our demo platfor
 - http://try.activeeon.com//cloud-automation[Add services^]
 
 We also provide REST and command line interfaces for advanced users.
+https://try.activeeon.com/rest/
```

Comment:
I would go for:

http://try.activeeon.com/rest[try.activeeon.com/rest^]

(similarly to line 19, just to hide the protocol)